### DISCUSSION TITLE
Is this a touch screen?

### DISCUSSION THREAD

**Discussion by RussNelson** [mathoudebine/turing-smart-screen-python#74] [Q&A] [answered]
Is this device also a touch screen? If so, do we know how to receive touch events? Where did information about the protocol come from, anyway?

**Comment by mathoudebine** [comment]
@RussNelson (converting this issue to a discussion)

**Comment by mathoudebine** [comment, marked as answer]
Hello @RussNelson this device has unfortunately no touchscreen (too expensive I think compared to other hardware choices made for this product...)
For the protocol, as there is no developer SDK or documentation, it comes from decompiling the original Windows software (made with Visual Studio) and transposing the protocol in Python. New Windows software have been mangled to forbid decompiling unfortunately...
